_i18n/pt.yml

```diff
@@ -21,7 +21,7 @@ home:
   how_we_work:
     title: Como trabalhamos
     description: |
-      Somos uma organização democraticamente estruturada, totalmente voluntária e liderada por trabalhadores. Nesse momento, a associação consiste em participar de reuniões pessoalmente e trabalhar em vários projetos nos quais as pessoas estão interessadas. Organizamos on-line, mas os encontros em vida real são o ponto crucial do que fazemos. Confira nosso <a href="/pt/guia-da-comunidade">Guia da comunidade</a>  para mais informações. Também entre no <a href="/subscribe">Slack</a> para se conectar com pessoas em sua região.
+      Somos uma organização democraticamente estruturada, totalmente voluntária e liderada por trabalhadores. Nesse momento, a associação consiste em participar de reuniões pessoalmente e trabalhar em vários projetos nos quais as pessoas estão interessadas. Organizamos on-line, mas os encontros em vida real são o ponto crucial do que fazemos. Confira nosso <a href="/pt/guia-da-comunidade/">Guia da comunidade</a>  para mais informações. Também entre no <a href="/subscribe">Slack</a> para se conectar com pessoas em sua região.
```

Comment:
I cannot figure out why, but `techworkerscoalition.org/pt/guia-da-comunidade/` does not work...